From e433374c73f0cf755b68e91a2760858ac6043aca Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Sat, 28 Jul 2018 19:03:57 +0800
Subject: [PATCH 4253/5242] MLK-19081 drm/bridge: sec-dsim: correct args of
 'drm_bridge_attach' call

commit  84e89aa94c2cac5b1ab3e00043949a6cc3348a98 from
https://source.codeaurora.org/external/imx/linux-imx.git

Obviously, according to the drm_bridge_attach()'s definition,
the passing arguments of its function call should be corrected
here.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/bridge/sec-dsim.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/bridge/sec-dsim.c b/drivers/gpu/drm/bridge/sec-dsim.c
index df76bf2..5c8bf95 100644
--- a/drivers/gpu/drm/bridge/sec-dsim.c
+++ b/drivers/gpu/drm/bridge/sec-dsim.c
@@ -395,7 +395,7 @@ static int sec_mipi_dsim_bridge_attach(struct drm_bridge *bridge)
 
 	dsim->next = next;
 	next->encoder = encoder;
-	ret = drm_bridge_attach(encoder, bridge, next);
+	ret = drm_bridge_attach(encoder, next, bridge);
 	if (ret) {
 		dev_err(dev, "Unable to attach bridge %s: %d\n",
 			remote->name, ret);
-- 
1.7.9.5

